app/code/Magento/SampleData/Test/Unit/Console/Command/AbstractSampleDataCommandTest.php

```diff
@@ -181,4 +181,19 @@ protected function setupMocks(
         array $sampleDataPackages,
         string $pathToComposerJson
     ) : array;
+
+    /**
+     * Expected arguments for `composer config` to set missing field "version"
+     *
+     * @return array
+     */
+    protected function expectedComposerArgumentsCommandConfig(): array
+    {
+        return [
+            'command' => 'config',
+            'setting-key' => 'version',
+            'setting-value' => ['0.0.1'],
+            '--quiet' => 1
+        ];
```

Comment:
That can be just `private const STUB_EXPECTED_COMPOSER_CONFIG`

Reply:
Hello @lbajsarowicz 
Thank you for your review.
I fixed it. Please check.
Thank you in advance.